docs/gitrepo-add.md

```diff
@@ -49,6 +49,12 @@ spec:
   #
   # clientSecretName: my-ssh-key
   #
+  # If fleet.yaml contains private helm repo that requires autheticate,
```

Comment:
> If fleet.yaml contains a private Helm repo that requires authentication,
 > provide the credentials in a K8s secret and specify them here. Details are provided
 > in the fleet.yaml documentation.